cloudfiles/interfaces.py

```diff
@@ -574,11 +586,20 @@ class S3Interface(StorageInterface):
   # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Bucket.delete_objects
   # claims batch size limit is 1000
   delete_batch_size = 1000
-  def __init__(self, path, secrets=None):
+  def __init__(self, path, secrets=None, user_project=None):
     super(StorageInterface, self).__init__()
     global S3_POOL
+
+    if user_project is None:
+      self._additional_attrs = {}
+    elif user_project == 'requester':
+      self._additional_attrs = {'RequestPayer': user_project}
+    else:
```

Comment:
This makes sense in context, but you should mention user_project otherwise people might not be sure what to fix.